lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(_options, args,  _config, endpoint),
+    do: with_authentication(endpoint, &do_delete(&1, args))
+
+  defp do_delete(_endpoint, []) do
+    display_arguments_error
+  end
+
+  defp do_delete(endpoint, relay_names) when is_list(relay_names) do
+    Enum.reduce_while(relay_names, :ok, fn relay_name, _acc ->
```

Comment:
Looks like we could conceivably delete a subset of the relays we've been given. In that case, it would be good to indicate to the user which ones were deleted and which ones weren't, instead of just saying "Something went wrong".